memfs/storage.go

```diff
@@ -220,6 +227,7 @@ func (c *content) ReadAt(b []byte, off int64) (n int, err error) {
 	}
 
 	btr := c.bytes[off : off+l]
+	c.m.RUnlock()
```

Comment:
shouldn't this go the end of the func? asking because, i _think_ `btr` would be a pointer to a sub slice of `c.bytes`?